src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/ExprEquipCompCameraOverlay.java

```diff
@@ -0,0 +1,106 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.bukkitutil.NamespacedUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.ValidationResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+import org.skriptlang.skript.log.runtime.SyntaxRuntimeErrorProducer;
+
+@Name("Equippable Component - Camera Overlay")
+@Description({
+	"The camera overlay for the player when the item is equipped.",
+	"Example: The jack-o'-lantern view when having a jack-o'-lantern equipped as a helmet.",
+	"The camera overlay is represented as a namespaced key.",
+	"A namespaced key can be formatted as 'namespace:id' or 'id'; "
+		+ "Can only contain one ':' to separate the namespace and the id, alphanumeric characters, periods, underscores, and dashes.",
```

Comment:
Just noting that it might be better to link to a tutorial page in the future. Also, I'd update this with my suggestion from `english.lang`